tests/core_tests/chaingen_main.cpp

```diff
@@ -50,6 +50,8 @@ namespace
 int main(int argc, char* argv[])
 {
   TRY_ENTRY();
+  static char monero_fast_pow[] = "MONERO_FAST_POW=1";
+  putenv(monero_fast_pow);
```

Comment:
IMO it would not hurt anybody if this was defaulted to be tested with an undefined MONERO_FAST_POW, and left for the individual user to enable it locally at their own will.